File: lib_base/src/main/java/com/czl/lib_base/annotation/TodoPriority.java
package com.czl.lib_base.annotation;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

/**
 * @author Alwyn
 * @Date 2021/1/6
 * @Description
 */
@IntDef({TodoPriority.HIGH,TodoPriority.NORMAL,TodoPriority.LOW})
@Retention(RetentionPolicy.SOURCE)
public @interface TodoPriority {
    int HIGH = 1;
    int NORMAL = 2;
    int LOW = 3;
}

File: lib_base/src/main/java/com/czl/lib_base/annotation/TodoPriority.kt
package com.czl.lib_base.annotation

import androidx.annotation.IntDef

/**
 * @author Alwyn
 * @Date 2021/1/6
 * @Description
 */
@IntDef(TodoPriority.HIGH, TodoPriority.NORMAL, TodoPriority.LOW)
@kotlin.annotation.Retention(AnnotationRetention.SOURCE)
annotation class TodoPriority {
    companion object {
        const val HIGH = 1
        const val NORMAL = 2
        const val LOW = 3
    }
}